cspell.json

```diff
@@ -1,18 +1,17 @@
 {
     "version": "0.1",
     "words": [
-        "ACMR",
```

Comment:
Please don't lowercase words. cSpell **is** case-sensitive: if it has only "ASTs" in the dictionary, it marks "asts" as an error.